identity/identity.go

```diff
@@ -185,20 +182,11 @@ func (i Identity) TableName(context.Context) string {
 	return "identities"
 }
 
-func (i *Identity) lock() *sync.RWMutex {
-	if i.l == nil {
-		i.l = new(sync.RWMutex)
-	}
-	return i.l
-}
```

Comment:
This was never working as it should, as the check `i.l == nil` and then the assignment is not atomic. Therefore the lock does not work for exclusive locking.
I did not find any reason for the lock in the first place, and only a hand full of methods actually used it, while a lot more should really. Therefore, I just removed it all together.